kolibri/plugins/management/assets/src/vue/user-page/user-edit-modal.vue

```diff
@@ -96,6 +96,35 @@
   const UserKinds = require('kolibri.coreVue.vuex.constants').UserKinds;
 
   module.exports = {
+    $trNameSpace: 'user-edit-modal',
+    $trs: {
+      modalTitle: 'Edit Account Info',
+      // input labels
+      fullName: 'Full Name',
+      username: 'Username',
+      userKind: 'User Kind',
+      enterNewPw: 'Enter new password',
+      confirmNewPw: 'Confirm new password',
```

Comment:
Yup, agreed. @DXCanas and I were talking about this yesterday. For this PR, feel free to make updates as-you-go but it's not the main point.

We can do another pass through the whole app with the specific goal of updating copy consistency. In the meantime, it's worth reading through the entire [writing style guide](https://material.io/guidelines/style/writing.html) because there are many other non-obvious conventions (e.g. I was surprised that they leave off periods in solitary sentences).